File: 프로그래머스/0/120811. 중앙값 구하기/중앙값 구하기.py
def solution(array):
    
    
    for i in range(len(array)):
        for j in range(len(array)-1-i):
         if(array[j]>array[j+1]):
            temp = array[j]
            array[j] = array[j+1]
            array[j+1] = temp
            
        
    return array[len(array)//2]

File: 프로그래머스/0/120811. 중앙값 구하기/중앙값 구하기.js
function solution(array) {
    let answer = 0;
    
    for(let i=0; i<array.length-1;i++){
        for(let k=i+1;k<array.length;k++){
            if(array[i] > array[k]){
                let temp = array[i];
                array[i] = array[k];
                array[k] = temp;
            }
        }
    }
    answer = array[Math.floor(array.length/2)];
    return answer;
}